controllers/sql.go

```diff
@@ -32,9 +32,12 @@ func ExecuteScriptQuery(rq *http.Request, queriesPath string, script string) ([]
 	}
 
 	sc := config.PrestConf.Adapter.ExecuteScriptsCtx(rq.Context(), rq.Method, sql, values)
-	if sc.Err() != nil {
+	if sc.Err() != nil && config.PrestConf.Debug {
 		err = fmt.Errorf("could not execute sql %v, %s", sc.Err(), sql)
```

Comment:
in this case you can just remove the sql from the err msg, this is fine since we dont want users to know what kind of SQL we're running.

```go
    return nil, errors.New("could not execute sql")
```

Reply:
@arxdsilva I was thinking of letting it show on the web, so developers using staging environments could see what is going on their server without tty access.